front/poke/temporal/activities.ts

```diff
@@ -537,8 +537,8 @@ export async function deleteWorkspaceActivity({
     });
     await FileResource.deleteAllForWorkspace(workspace, t);
     await DataSourceViewResource.deleteAllForWorkspace(auth, t);
-    await VaultResource.deleteAllForWorkspace(auth, t);
     await GroupResource.deleteAllForWorkspace(workspace, t);
+    await VaultResource.deleteAllForWorkspace(auth, t);
```

Comment:
After looking quickly at the code, I don't think this will fix it. Let me explain why and you tell me if that makes sense: 

When we call destroy for either group or vault resource, we delete both the model (group or vault) + the associated GroupVault objects. 

I think our problem here is `deleteAllForWorkspace` does not do it. 

So I think the fix would be to edit both `GroupResource` and `VaultResource` to make sure it deletes both the model and the `GroupVaultModel`. (As we do in the destroy function of the resource).

But it's very weird that you can't reproduce the issue locally tho 😕